Services/DataX.Flow/Flow.InteractiveQueryService/Controllers/InteractiveQueryController.cs

```diff
@@ -130,14 +131,14 @@ public async Task<ApiResult> DeleteKernels()
 
         [HttpPost]
         [Route("kernels/deleteall")]
-        public async Task<ApiResult> DeleteAllKernels()
+        public async Task<ApiResult> DeleteAllKernels([FromBody]string databricksToken)
         {
             try
             {
                 RolesCheck.EnsureWriter(Request);
 
                 InteractiveQueryManager iqm = new InteractiveQueryManager(_logger, _configuration);
-                return await iqm.DeleteAllKernels().ConfigureAwait(false);
+                return await iqm.DeleteAllKernels(databricksToken).ConfigureAwait(false);
```

Comment:
any way to ensure any future api have this, some overloads?  maybe not in this pr.